app/views/sessions/new.html.haml

```diff
@@ -4,7 +4,10 @@
       $('#nick').focus();
     });
 
-- title t('.title')
+- if FoodsoftConfig[:instance_name].present?
+  - title FoodsoftConfig[:instance_name]
```

Comment:
why not `FoodsoftConfig[:instance_name] || t('.title')` to avoid duplication of the config access?

what's the benefit of `instance_name` over the already existing `name`?

Reply:
There is no benefit if every foodcoop changes their foodcoop's name at the configuration page. Actually a lot of foodcoops didn't do it. But maybe this is a special case at our hosting platform.
As you suggested it would be better to use `FoodsoftConfig:[:name]` to make it editable by users.